table/block_based_table_reader.h

```diff
@@ -612,6 +619,10 @@ class BlockBasedTableIterator : public InternalIterator {
   // TODO use block offset instead
   std::string prev_index_value_;
   const SliceTransform* prefix_extractor_;
+  // If the blocks over which we iterate are index blocks
+  bool is_index_;
+  // If the keys in the blocks over which we iterate include 8 byte sequence
+  bool key_includes_seq_;
```

Comment:
If no special reason, I would put it around other bools, so that they will be free from extra memory.